core/cache/inMemoryBackend_test.go

```diff
@@ -0,0 +1,17 @@
+package cache_test
+
+import (
+	"testing"
+
+	"flamingo.me/flamingo/v3/core/cache"
+)
+
+func Test_RunDefaultBackendTestCase_InMemoryBackend(t *testing.T) {
+	backend := cache.NewInMemoryCache()
+
+	testcase := &cache.BackendTestCase{
```

Comment:
> Like the idea with the Testing Lib.
> * Some go packages add this to a subpackage "testlibs" - I found no official recommendation
moving the the file to an sub-package would case an import cycle, because the backend interface is needed there
> * We can even add a Factory Method to avoid missuse:  NewBackendTest(t,backend)
makes sense i will change